src/object.d

```diff
@@ -3257,8 +3252,6 @@ private size_t getArrayHash(in TypeInfo element, in void* ptr, in size_t count)
 }
 
 
-// @@@BUG5835@@@ tests:
-
```

Comment:
It's not that there's something special, it's a link back to the impetus to create it. For example, if I looked at the unit tests for 10574, I would have no idea why someone added those tests. The comment allows me to avoid searching for it.

A comment is there to help solidify the reason for having it. Indeed, this unit test is confusing without the context of the comment.

BTW, why are the asserts marked `//fails` ? Clearly they don't fail, since they are passing unit tests!